tensorflow/python/framework/ops_test.py

```diff
@@ -255,7 +255,9 @@ def testInputsAndOutputs(self):
     """, op3.node_def)
 
   def testDevicePresent(self):
-    op = ops.Operation(ops._NodeDef("None", "myop", device='/job:goo/device:GPU:0'), ops.Graph(), [], [])
+    op = ops.Operation(
+      ops._NodeDef("None", "myop", device='/job:goo/device:GPU:0'), ops.Graph(),
```

Comment:
Line continuation indent should be 4 spaces

Reply:
Done